HW1/main.swift

```diff
@@ -7,4 +7,256 @@
 
 import Foundation
 
-print("Hello, !")
+//1. Сортировка чисел: Дан массив чисел. Напишите функцию, которая возвращает отсортированный массив в порядке возрастания без использования стандартной функции sort().
+func sort(_ array: [Int]) -> [Int] {
+    var arr = array
+    let n = array.count
+    
+    for i in 0..<n-1 {
+        for m in 0..<n-i-1 {
+            if arr[m] > arr[m+1] {
+                let temp = arr[m]
+                arr[m] = arr[m+1]
+                arr[m+1] = temp
+            }
+        }
+    }
+    return arr
+}
+
+let a = sort([55, 4, 33, 2, 11])
+print(a)
+
+//2. Проверка палиндрома: Напишите функцию, которая принимает строку и возвращает true, если строка является палиндромом, и false в противном случае.
+func isPalindrome(string:String) -> Bool {
+    let reverseString = String(string.reversed())
+    
+    if string != "" && string == reverseString {
+        print("Это слово полиндром")
+        return true
+    } else {
+        print("Это не полиндром")
+        return false
+    }
+}
+
+//isPalindrome(string: "mam")
+
+//3. 3. Сумма элементов массива: Дан массив целых чисел. Напишите функцию, которая возвращает сумму всех элементов массива.
+func sumOf(numbers: [Int]) -> Int {
+    var sumNumber = 0
+    
+    for i in 1...numbers.count {
```

Comment:
for i in 1...numbers.count-1
```suggestion
    for i in 1...numbers.count-1 {
```